clusters/base/falco-sci.yaml

```diff
@@ -0,0 +1,536 @@
+            "w": 5,
+            "x": 0,
+            "y": 1
+          },
+          "id": 8,
+          "options": {
+            "code": {
+              "language": "plaintext",
+              "showLineNumbers": false,
+              "showMiniMap": false
+            },
+            "content": "# [Energy (E)](https://sci-guide.greensoftware.foundation/E/)\n\n### What are the different ways to  calculate energy?​\n\nSoftware systems cause emissions through the hardware that they operate on, both through the energy that the physical hardware consumes and the emissions associated with manufacturing the hardware.\n\nMore info [here](https://sci-guide.greensoftware.foundation/E/).",
+            "mode": "markdown"
+          },
+          "pluginVersion": "10.2.3",
+          "title": "Energy (E)",
+          "transparent": true,
+          "type": "text"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "prometheus"
+          },
+          "gridPos": {
+            "h": 10,
+            "w": 5,
+            "x": 5,
+            "y": 1
+          },
+          "id": 9,
+          "options": {
+            "code": {
+              "language": "plaintext",
+              "showLineNumbers": false,
+              "showMiniMap": false
+            },
+            "content": "# [Emissions (I)](https://sci-guide.greensoftware.foundation/I)\n\nThe carbon intensity of electricity is a measure of how much carbon (CO2eq) emissions are produced per kilowatt-hour (kWh) of electricity consumed.\n\nThe only value that matters if you’re trying to optimise the scheduling of your computing in real-time is the marginal emissions intensity. This is the emissions intensity of the marginal power plant which will be turned up if you schedule some computing at that moment.\n\nMore info [here]().",
+            "mode": "markdown"
+          },
+          "pluginVersion": "10.2.3",
+          "title": "Emissions (I)",
+          "transparent": true,
+          "type": "text"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "prometheus"
+          },
+          "gridPos": {
+            "h": 10,
+            "w": 5,
+            "x": 10,
+            "y": 1
+          },
+          "id": 10,
+          "options": {
+            "code": {
+              "language": "plaintext",
+              "showLineNumbers": false,
+              "showMiniMap": false
+            },
+            "content": "# [Embodied (M)](https://sci-guide.greensoftware.foundation/M/)\n\nEmbodied carbon (also known as embedded carbon) is the amount of carbon emitted during the creation and disposal of a hardware device.\n\nWhen software runs on a device, a fraction of the total embodied emissions of the device is allocated to the software. This is the value of M that you need to calculate in the SCI equation.\n\nMore info [here](https://sci-guide.greensoftware.foundation/M/).",
```

Comment:
I wrote docs for M in #73

How about we link to our docs here and link to the SCI guide from our docs?